aria-practices.html

```diff
@@ -2151,6 +2151,43 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
       </section>
     </section>
 
+    <section class="widget" id="meter">
```

Comment:
This section will conflict with #1189, but I can add anything from here to that PR if desired, or we can figure out another way to merge them

Reply:
@smhigley it seems like #1189 for design pattern, not the example. Did I misunderstand?

Reply:
It looks like the changes in this file have already been removed from the PR 👍